doc/adr/0003-cli-using-noun-verb-format.md

```diff
@@ -0,0 +1,41 @@
+# 3. CLI using noun verb format
+
+Date: 2021-07-09 
+
+Decsion made prior to this date
+
+## Status
+
+Accepted
+
+## Context
+
+### Glossary
+* **Command** - Action to perform, e.g., get, install, add, help
+* **Ojbect type** - Classification of types which the CLI will operate on
+
+We require consistency in the weave-gitops cli `wego` along with the ability to extend the object types and commands available.  For example, wego, wraps the flux ui and exposes it via the `wego flux` command.  We want a standardized way to enhance and augment the CLI without always requiring additional discussions.
+
+## Decision
+
+The CLI commands will be the nonu verb format.  
```

Comment:
```suggestion
The CLI commands will be the noun verb format.  
```

Reply:
Thanks for catching this.  I fixed this along with a few other items.